packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -135,7 +136,7 @@ async function initProject(appName: string, options: CLIOptions) {
       isContentfulTemplate(templateSource) &&
       normalizedOptions.deliveryFunction
     ) {
-      await cloneDeliveryFunction(fullAppFolder, !!normalizedOptions.typescript);
+      await cloneDeliveryFunction(fullAppFolder, !normalizedOptions.javascript);
```

Comment:
this change confuses me 🙃

Reply:
It's needed to make the Typescript a default option.
The function signature is like that: `cloneDeliveryFunction(destination: string, templateIsTypescript: boolean)`
If we omit both flags I want Typescript by default, hence the change. Do you suggest to write it in a different way?

Reply:
but this would use javascript as default? Doesn't it make sense something like `normalizedOptions.javascript ?? true`

Reply:
I think it would be good if I change the function from 
`export async function cloneDeliveryFunction(destination: string, templateIsTypescript: boolean)` 
to 
`export async function cloneDeliveryFunction(destination: string, templateIsJavascript: boolean)` 
to make it less confusing?

Reply:
Makes it even easier :)

Reply:
Done ;)

Reply:
lol ty for helping my peanut brain with this :)